CMakeLists.txt

```diff
@@ -337,13 +338,15 @@ add_executable(roboteam_ai
         ${DEMO_SOURCES}
         ${INTERFACE_SOURCES}
         ${WORLD_SOURCES}
-        )
+        src/treeinterp/MidFieldHarassRole.cpp include/roboteam_ai/treeinterp/MidFieldHarassRole.h src/TreeHelper/BeingPassedToHelper.cpp include/roboteam_ai/TreeHelper/BeingPassedToHelper.h src/TreeHelper/RobotOutOfFieldHelper.cpp include/roboteam_ai/TreeHelper/RobotOutOfFieldHelper.h src/treeinterp/CoachDefenderRole.cpp include/roboteam_ai/treeinterp/CoachDefenderRole.h)
```

Comment:
Please don't do this, but add them to one of the ${SOURCES} variables such as BT or whereever they could be relevant.

Reply:
And seperate them onto multiple lines.